boards/opentitan/earlgrey-cw310/.cargo/config.toml

```diff
@@ -2,5 +2,19 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 # Copyright Tock Contributors 2022.
 
+include = [
+  "../../../cargo/tock_flags.toml",
+  "../../../cargo/unstable_flags.toml",
+  "../../../cargo/riscv_flags.toml",
+  "../../../cargo/virtual_function_elimination.toml",
+  "../../../cargo/panic_abort_tests.toml",
+]
+
+[build]
+target = "riscv32imc-unknown-none-elf"
+
 [target.'cfg(target_arch = "riscv32")']
 runner = "./run.sh"
```

Comment:
Similarly, it would be great to comment why this exists here.

Reply:
I mean, that is already there.

Reply:
FWIW, this is a super standard Cargo thing. We can comment, but it basically invokes the `run.sh` command in the earlgray crate directory when running `cargo run` (in turn, this is script basically runs the opentitan qemu).